instrumentation/opentelemetry-instrumentation-urllib3/tests/test_urllib3_metrics.py

```diff
@@ -102,7 +111,9 @@ def test_basic_metrics(self):
             SpanAttributes.HTTP_SCHEME: "http",
         }
 
-        self.assertEqual(client_duration.name, "http.client.duration")
+        self.assertEqual(
+            client_duration.name, MetricInstruments.HTTP_CLIENT_DURATION
```

Comment:
Have to say I prefer to assert the exact string instead of a variable that can change its value over time

Reply:
Me too, and that's the way I did in the first version of the tests. But while writing it I thought referencing the variable from semconv package would  make better to notice the changes from old to new in metric names

Reply:
But I'm also fine with using the exact name

Reply:
We probably should rewrite all the sem conv tests to assert exact strings similar to [here](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2746#discussion_r1693506236). Can start with this pr and we can leave the other changes to other prs.

Reply:
Agreed. So while we are here I'll change the PR to be the refactor for urllib3 tests then to use plain string instead of referencing semconv symbols